tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -3355,6 +3355,72 @@ namespace EnergyPlus {
 			} ) );
 
 		}
+		TEST_F (SQLiteFixture, OutputProcessor_GenOutputVariablesAuditReport)
+		{
+			std::string const idf_objects = delimited_string ({
+				"Output:Variable,*,Site Outdoor Air Drybulb Temperature,timestep;",
+				"Output:Variable,*,Boiler Gas Rate,detailed;",
+				"Output:Variable,*,Boiler Heating Rate,detailed;",
+				"Output:Meter,Electricity:Facility,timestep;",
+			});
+
+			ASSERT_FALSE (process_idf (idf_objects));
+
+			DataGlobals::DayOfSim = 365;
+			DataGlobals::DayOfSimChr = "365";
+			DataEnvironment::Month = 12;
+			DataEnvironment::DayOfMonth = 31;
+			DataEnvironment::DSTIndicator = 0;
+			DataEnvironment::DayOfWeek = 3;
+			DataEnvironment::HolidayIndex = 0;
+			DataGlobals::HourOfDay = 24;
+			DataGlobals::NumOfDayInEnvrn = 365;
+			DataGlobals::MinutesPerTimeStep = 10;
+
+			if ( DataGlobals::TimeStep == DataGlobals::NumOfTimeStepInHour ) {
+				DataGlobals::EndHourFlag = true;
+				if ( DataGlobals::HourOfDay == 24 ) {
+					DataGlobals::EndDayFlag = true;
+					if ( (!DataGlobals::WarmupFlag) && (DataGlobals::DayOfSim == DataGlobals::NumOfDayInEnvrn) ) {
+						DataGlobals::EndEnvrnFlag = true;
+					}
+				}
+			}
+
+			if ( DataEnvironment::DayOfMonth == WeatherManager::EndDayOfMonth (DataEnvironment::Month) ) {
+				DataEnvironment::EndMonthFlag = true;
+			}
+
+			TimeValue.allocate (2);
+
+			auto timeStep = 1.0 / 6;
+
+			SetupTimePointers ("Zone", timeStep);
+			SetupTimePointers ("HVAC", timeStep);
+
+			TimeValue (1).CurMinute = 50;
+			TimeValue (2).CurMinute = 50;
+
+			EnergyPlus::sqlite = std::move (sqlite_test);
```

Comment:
@JasonGlazer Make sure you `std::move` back to `sqlite_test`  from `EnergyPlus::sqlite` after you are finished running any functions that require `EnergyPlus::sqlite` to exist.

`sqlite_test = std::move( EnergyPlus::sqlite );`